docs/virtualbox.md

```diff
@@ -94,7 +94,7 @@ sudouser@host:~$ pip install remotevbox --user
 sudouser@host:~$ python -c 'import remotevbox; vbox = remotevbox.connect("http://<external-ip>:18083", "", ""); print(vbox.get_version()); print(vbox.list_machines()); vbox.disconnect()'
 ```
 
-The latest command should return something like the following:
+The last command should return something like the following:
```

Comment:
I have seen some python modules that also install a CLI when you pip install.  I might be cool for your remotevbox to include that?  Not necessary at all, just a suggestion.

Reply:
Hmm, that would be cool, thanks for an idea: ilyaglow/remote-virtualbox#10